### ISSUE TITLE
Usability testing for datatables: all receipts, and indiv. contrib

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-testing#120]
**User story:** As a data user, I need to be able to increase my time period selection filter larger than a single 2 year time period for receipts and disbursements data so that I can view and filter data for that expanded time span.

**What we're after:** Following the implementation of the MVP 2-year restriction lifting and front-end implementation of the new time filter interaction on the all receipts and individual contributions datatables, we want to conduct usability testing to identify any pain points and any breaks that users might experience. This testing can be done once the work is merged into feature or stage. 

Because we're changing the data that users will see on first load, as well as how they conduct their own searches and the performance issues that may exist, we need to determine what those specific problems might be and if there are new performance issues that must be addressed and whether the new time filter interaction helps or hinders that search. 

**Completion criteria:**
- [x] Draft scheduling emails and identify timeslots for usability testing
- [x] Identify and contact users for testing
- [x] ~Build prototype in Invision~ using feature: https://fec-feature-cms.app.cloud.gov/
- [x] Write test script
- [x] Conduct testing
- [x] Summarize findings and compile results in mini-presentation
- [ ] ~Make design changes based on findings~
- [x] Present at demo

**Comment by JonellaCulmer** [comment]
Test script: https://docs.google.com/document/d/1oyYf-pf2oFV0jBfjk-KKTiceCO6Ldh7zcnsLCiObN-Y/edit#

**Comment by JonellaCulmer** [comment]
Findings https://app.mural.co/t/fec4461/m/fec4461/1561740259151/aa1efb4bd63bb255db5d49837e9ffeaf17baaf42

**Comment by JonellaCulmer** [comment]
Related issues/PRs resulting from usability testing:
- https://github.com/fecgov/openFEC/issues/3845
- https://github.com/fecgov/fec-cms/issues/2987
- https://github.com/fecgov/openFEC/issues/3775
- https://github.com/fecgov/fec-cms/pull/2511
